src/inference/src/os/lin/lin_system_conf.cpp

```diff
@@ -188,9 +91,155 @@ struct CPU {
                 _cores = _processors;
             }
         }
+        std::vector<std::vector<std::string>>().swap(system_info_table);
     }
 };
 static CPU cpu;
+
+void parse_processor_info_linux(const int _processors,
```

Comment:
Not necessary for this PR, but nice to have in future. It is better to explain what structure `system_info_table` table has to simplify future maintenance of `parse_processor_info` both for linux and windows.

Reply:
So far looks like we cannot use unified `system_info_table` for both Linux and Windows. 

We would implement separated parser and test cases for Linux and Windows. Then let's review and check if we can merge them.